scripts/bootstrap.sh.in

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+set -e
+set -u
+if test -z "$(type -p)"; then set -o pipefail; fi
+
+RPM=${RPM:-$(command -v rpm)}
+SYSTEMCTL=${SYSTEMCTL:-$(command -v systemctl)}
+YUM=${YUM:-$(command -v yum)}
+SALT_CALL=${SALT_CALL:-salt-call}
+
+SALT_MINION_FILE_CLIENT_LOCAL_CONF=/etc/salt/minion.d/99-file-client-local.conf
+
+die() {
+        >/dev/stderr echo "$@"
+        exit 1
+}
+
+pre_minion_checks() {
+        test "x$(whoami)" = "xroot" || die "Script must run as root"
+        test -n "${RPM}" || die "rpm not found"
```

Comment:
why not `-x` directly? It also test the existence.

For better error messages?

Reply:
Yes, for error message purposes. There's a `-x` below.